config/initializers/sentry.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+Sentry.init do |config|
+  config.dsn = 'https://e8bd9a43b666488db6bee4975626669d@o949860.ingest.sentry.io/5898547'
```

Comment:
configs with secret keys should not be public :) Try to play with [ENV](https://www.rubyguides.com/2019/01/ruby-environment-variables/)

Reply:
thanks, I added dotenv gem and made the key private :)